pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -249,6 +249,11 @@ type KubeletConfiguration struct {
 	// Default: "10s"
 	// +optional
 	CPUManagerReconcilePeriod metav1.Duration `json:"cpuManagerReconcilePeriod,omitempty"`
+	// Map of QoS resource reservation percentages (memory only for now).
+	// Requires the QOSReserved feature gate to be enabled.
+	// Default: nil
+	// +optional
+	QOSReserved map[string]string
```

Comment:
@sjenning This still needs `` `json:"qosReserved,omitempty"` ``

Reply:
opened follow on PR to fix this up https://github.com/kubernetes/kubernetes/pull/62925